tests/system-test/3-connectors/c#/TDengineDriver/TDengineDriver.cs

```diff
@@ -119,6 +119,7 @@ public struct TAOS_BIND
         public IntPtr error;
         public Int64 u;
         public uint allocated;
+        public bool is_string;
     }
 
```

Comment:
The struct need to keep as same as taosc's  struct. Cannot add new field "is_string"